patch/kconfig-inclusions

```diff
@@ -126,6 +126,56 @@ CONFIG_SENSORS_EMC2305=m
 [marvell-arm64]
 CONFIG_MARVELL_AC5_WATCHDOG=m
 
+[pensando]
+CONFIG_ARCH_PENSANDO=y
+CONFIG_ARCH_PENSANDO_ELBA_SOC=y
+CONFIG_PENSANDO_SOC_CAPMEM_HUGEPAGE=y
+CONFIG_PENSANDO_SOC_PENFW=y
+CONFIG_EDAC_ELBA=y
+CONFIG_IRQ_PENSANDO=y
+CONFIG_MFD_PENSANDO_ELBASR=y
+CONFIG_RESET_ELBASR=y
+CONFIG_UIO_PCIEMAC=m
+CONFIG_UIO_PENMSI1=m
+CONFIG_UIO_PENGIC=m
+CONFIG_UIO_PENMSI=m
+CONFIG_ARM64_VA_BITS_39=y
+CONFIG_COMMON_CLK_CS2000_CP=y
+CONFIG_CONTEXT_TRACKING=y
+CONFIG_CRYPTO_MANAGER_DISABLE_TESTS=y
+CONFIG_DEVTMPFS_MOUNT=y
+CONFIG_EMBEDDED=y
+CONFIG_GENERIC_IRQ_DEBUGFS=y
+CONFIG_GPIO_DWAPB=y
+CONFIG_I2C_RD1173=y
+CONFIG_IP_PNP=y
+CONFIG_IP_PNP_BOOTP=y
+CONFIG_IP_PNP_DHCP=y
+CONFIG_IRQSOFF_TRACER=y
+CONFIG_KALLSYMS_ALL=y
+CONFIG_MEMTEST=y
+CONFIG_MMC_DEBUG=y
+CONFIG_MMC_SDHCI_CADENCE=m
+CONFIG_NO_HZ_FULL=y
+CONFIG_PENSANDO_SOC_CAPMEM_HUGEPAGE=y
+CONFIG_PENSANDO_SOC_PENFW=y
+CONFIG_PMBUS=y
+CONFIG_PREEMPTIRQ_TRACEPOINTS=y
+CONFIG_RCU_NOCB_CPU=y
+CONFIG_RING_BUFFER_ALLOW_SWAP=y
+CONFIG_RTC_DRV_PCF85363=y
+CONFIG_SCHED_TRACER=y
+CONFIG_SENSORS_LTC2978=y
+CONFIG_SENSORS_PMBUS=y
+CONFIG_SENSORS_TPS53679=y
+CONFIG_SPI_CADENCE_QUADSPI=m
+CONFIG_SPI_DESIGNWARE=m
+CONFIG_SPI_DW_MMIO=m
+CONFIG_SQUASHFS_DECOMP_MULTI=y
+CONFIG_TRACER_SNAPSHOT_PER_CPU_SWAP=y
+CONFIG_VIRT_CPU_ACCOUNTING=y
+CONFIG_VIRT_CPU_ACCOUNTING_GEN=y
```

Comment:
What's the purpose of the following configs? Can they be removed?

```
CONFIG_CRYPTO_MANAGER_DISABLE_TESTS=y
CONFIG_VIRT_CPU_ACCOUNTING_GEN=y
CONFIG_MMC_DEBUG=y
CONFIG_GENERIC_IRQ_DEBUGFS=y
CONFIG_NO_HZ_FULL=y
```

Reply:
```CONFIG_NO_HZ_FULL, CONFIG_VIRT_CPU_ACCOUNTING_GEN```(required by CONFIG_NO_HZ_FULL) -> This is required for vpp. We have below options passed from u-boot nohz_full=1,2,3,5,6,7,9,10,11,13,14,15 rcu_nocbs=1,2,3,5,6,7,9,10,11,13,14,15 rcu_nocb_poll
```CONFIG_MMC_DEBUG, CONFIG_GENERIC_IRQ_DEBUGFS``` -> We faced few issues with eMMC from one of the vendor in the field. This is required for debugging.
```CONFIG_CRYPTO_MANAGER_DISABLE_TESTS``` is the Linux kernel default which disables crypto-self tests. Any reason for overriding the default in current SONiC kernel ?

Reply:
> `CONFIG_CRYPTO_MANAGER_DISABLE_TESTS` is the Linux kernel default which disables crypto-self tests. Any reason for overriding the default in current SONiC kernel ?

Looks like Debian disables this config (causing self tests to be run), but Ubuntu (for example) keeps it enabled, and our configs are primarily based on Debian's configs; I'm fine with it either way.

Reply:
if we plant to disable it, why make it pensando specific, we should move them to arm64

Reply:
Updated.